enterprise-suite/gotests/Makefile

```diff
@@ -0,0 +1,33 @@
+files := $(shell find . -path ./vendor -prune -o -name '*.go' -print)
+
+.PHONY: all
+all: check
+
+unformatted = $(shell goimports -l $(files))
+
+.PHONY: check
+check: checkformat vet lint
+
+.PHONY: checkformat
+checkformat:
+	$(if $(unformatted), \
+	@echo "needs formatting: $(unformatted)" && \
+	echo "run make format" &&\
+	exit 1 \
+	)
+
+.PHONY: vet
+vet:
+	go vet ./...
+
+.PHONY: lint
+lint:
+	golint -set_exit_status $(go list ./...)
+
+.PHONY: go-tests
+run-tests:
+	ginkgo -r -- --namespace=$(TILLER_NAMESPACE) --tiller-namespace=$(TILLER_NAMESPACE)
```

Comment:
just double checking that TILLER_NAMESPACE is intended to be used for two namespace things here..

Reply:
It's a little confusing - it should install in the same namespace both things. I'll change it...